IMDA-BTG/aiverify
# Fix/missing dist stock plugins and wrong fmt widget

## Description

Update .gitignore to remove "*.zip" line and add plugin zips for process checklist and report plugins.
Fix wrong fmt widget included in the aiverify report template for classification

## Motivation and Context
Generated plugin zip are not being add to git repo because of the line *.zip in .gitignore
Aiverify report template for classification contained fmt-r instead of fmt-c


## Type of Change













## How to Test

Check dist folder under plugin project "aiverify.stock.process-checklist"  to verify that the plugin zip exists. Install new zip to portal and verify can install.

Repeat above process for plugin "aiverify.stock.reports"

Check that AI Verify report template for classification models has FMT-C as the test to run instead of FMT-R

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]
